plugins/download.py

```diff
@@ -219,7 +221,7 @@ def _get_source_packages(pkgs):
 
     def _get_query(self, pkg_spec):
         """Return a query to match a pkg_spec."""
-        if os.path.exists(pkg_spec):
+        if pkg_spec.endswith('.rpm') and os.path.exists(pkg_spec) and os.path.isfile(pkg_spec):
```

Comment:
why not `os.path.exists(pkg_spec) and os.path.isfile(pkg_spec)` -> `os.path.isfile(pkg_spec)`?